en/cocos-service/agc-applinking.md

```diff
@@ -118,22 +121,22 @@ Most of HUAWEI Services need the `agconnect-services.json` configuration file. I
   huawei.agc.applinking.appLinkingService.on(huawei.agc.applinking.AGC_APP_LINKING_EVENT_LISTENER_NAME.RECEIVE_LINK_CALLBACK, (data) => {
       console.log("getData, data.code =", data.code);
       if (data.code === 1) {
-          this.console.log('receive deepLink：', data.getDeepLink());
+          this.console.log('receive deepLink:', data.getDeepLink());
       }
   }, this);
   ```
 
 - [Publish to the Android platform](../publish/publish-native.md) after the code is added. Please make sure that the **Package Name** on the **Build** panel is consistent with the **Package Name** set in the AppGallery Connect console.
 
-- Refer to AGC - AppLinking [Creating a Link in AppGallery Connect](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-Guides/agc-applinking-createlinks-byagc) documentation, create a App Linking, select **Open in your Android app** option of **Link behavior for Android**, and choose current application in setting box below.
+- Refer to [Creating a Link in AppGallery Connect](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-Guides/agc-applinking-createlinks-byagc) documentation, create a App Linking, then select **Open in your Android app** option of **Link behavior for Android**, and choose current application in setting box below (if you have not created an application, you can click the **Add Android app** button to create).
```

Comment:
Refer to the [Creating a Link in AppGallery Connect]() documentation to create an App Linking. When setting the Android Linking behavior, select **Open in your Android app** and select the current app in the settings box below......